services/centralized-grafana/48.3.1/centralized-grafana.yaml

```diff
@@ -11,7 +11,7 @@ spec:
         kind: HelmRepository
         name: mesosphere.github.io-charts-staging
         namespace: kommander-flux
-      version: 48.3.1
+      version: 48.3.2
```

Comment:
not sure if this bump was intentional but the latest version is actually 48.3.3 https://github.com/mesosphere/charts/blob/6135b0df4858e916d6bd5705ca63a43c885153bf/staging/kube-prometheus-stack/Chart.yaml#L24

Reply:
no it was a mistake, thanks for your review.